tests/pipelines/test_vote.py

```diff
@@ -239,13 +239,13 @@ def test_valid_block_voting_with_transfer_transactions(monkeypatch, b):
     vote_pipeline = vote.create_pipeline()
     vote_pipeline.setup(indata=inpipe, outdata=outpipe)
 
+    vote_pipeline.start()
     inpipe.put(block)
+    import time; time.sleep(1)
```

Comment:
@vrde We probably don't want this, but I don't have a good alternative.  The voter module is running in different processes, and if one block with the fulfillment `tx` doesn't get written first, this will (correctly) raise an exception.